src/video.cpp

```diff
@@ -1281,6 +1281,17 @@ void encode_run(
   const encoder_t &encoder,
   void *channel_data) {
 
+  // get minimum fps from config
+  auto MIN_FPS = std::chrono::milliseconds(*&config::video.min_fps * 10);
+
+  // ensure MIN_FPS is at least 10 and less than 2400 milliseconds
+  if(MIN_FPS < std::chrono::milliseconds(10)) {
```

Comment:
This should be changed to verify that minimum_frametime is at least 100 milliseconds.

Also we probably want to make sure the user didn't exceed the framerate of the config as well.

Reply:
The formula you gave calculates to 16.7 for 60 fps... so I think we need to go lower than 100.

Reply:
The condition check would basically be

if the milliseconds is greater than 100 milliseconds or lower than the milliseconds of the streams framerate, set it to 100 milliseconds and log about it.

we don’t have to worry about multiple sessions, because this code already is broken for that (it’s the main reason why the bug about multiple streams all share the same framerate)